tests/bugzilla/Bugzilla1266.java

```diff
@@ -74,52 +94,58 @@ public void execute(Datalet d)
+                factory = TransformerFactory.newInstance();
+                logger.logMsg(Logger.STATUSMSG, "About to factory.newTemplates(" + QetestUtils.filenameToURL("identity.xsl") + ")");
+                templates = factory.newTemplates(new StreamSource(QetestUtils.filenameToURL("identity.xsl")));
+                transformer = templates.newTransformer();
+
+                // Set the errorListener and validate it
+                transformer.setErrorListener(loggingErrorListener);
+                if (transformer.getErrorListener() == loggingErrorListener)
+                    logger.checkPass("set/getErrorListener on transformer");
+                else {
+                    logger.checkFail("set/getErrorListener on transformer");
+		    throw new Exception("failed set/getErrorListener on transformer");
+                }
+
+                logger.logMsg(Logger.STATUSMSG, "Reproduce Bugzilla1266 - warning due to bad output props not propagated");
+                logger.logMsg(Logger.STATUSMSG, "transformer.setOutputProperty(encoding, illegal-encoding-value)");
+                transformer.setOutputProperty("encoding", "illegal-encoding-value");
+
+                logger.logMsg(Logger.STATUSMSG, "about to transform(...)");
+                transformer.transform(new StreamSource(QetestUtils.filenameToURL("identity.xml")), 
+                                      new StreamResult("Bugzilla1266.out"));
+                logger.logMsg(Logger.STATUSMSG, "after transform(...)");
+                logger.logMsg(Logger.STATUSMSG, "loggingErrorListener after transform:" + loggingErrorListener.getQuickCounters());
+
+                // Validate that one warning (about illegal-encoding-value) should have been reported
+                int[] errCtr = loggingErrorListener.getCounters();
+                            
+                if (errCtr[LoggingErrorListener.TYPE_WARNING] > 0)
+                    logger.checkPass("At least one Warning listned to for illegal-encoding-value");
+                else {
+                    logger.checkFail("No Warning reported by listener for illegal-encoding-value");
+		    throw new Exception("No Warning reported by listener for illegal-encoding-value");
+                }
                 
-            // Validate the actual output file as well: in this case, 
-            //  the stylesheet should still work
-            CheckService fileChecker = new XHTFileCheckService();
-            fileChecker.check(logger, 
-                    new File("Bugzilla1266.out"), 
-                    new File("identity.gold"), 
-                    "transform of good xsl w/bad output props into: " + "Bugzilla1266.out");
+                // Validate the actual output file as well: in this case, 
+                //  the stylesheet should still work
+                CheckService fileChecker = new XHTFileCheckService();
+                fileChecker.check(logger, 
+                                  new File("Bugzilla1266.out"), 
```

Comment:
This indent seems excessive

Reply:
One common indentation style for multi-line parameters, as seen earlier in the transformer.transform() call, is to align the parameters. As seen in the web view of this comment, that is in fact what's happening. Indentation has been done with spaces rather than tabs (safer when some editors allow setting the tab stops), but the style is consistent; I'm not sure why you're flagging this one. Clarify, please?

Reply:
When I look at this in my editor, indentation appears correct in context (aligned with the '}' above it) with or without untabifying. I'm not at all sure why git is flagging this as an indentation change, unless this was indentation that was previously missing. I'm open to suggestions re how to make git happier with it, but I'm really not seeing any there there.

Reply:
Whether it's new or not I see a 17 space indent here, which is way too much. I think I see why: it's trying to line up the arguments. If spotless is doing this, then that's another reason to hate on spotless, but for now not a blocker. If it's your editor please fix that before committing.

Reply:
Sigh. Code style conventions. 

This is exactly the same style (vertical alignment of arguments) used elsewhere in that same .java file. If we're gonna object to it, we should be reformatting the entire testcase to whatever our established preferred style is.